securing_apps/topics/oidc/fapi-support.adoc

```diff
@@ -13,6 +17,23 @@ To make sure that your clients are FAPI compliant, you can configure Client Poli
 and link them to the global client profiles for FAPI support, which are automatically available in each realm. You can use either `fapi-1-baseline` or `fapi-1-advanced` profile based on which FAPI
 profile you need your clients to conform with.
 
+In case you want to use link:{adminguide_link}#_oidc_clients[Pushed Authorization Request (PAR)], it is recommended that your client use
+both `fapi-1-baseline` profile and `fapi-1-advanced` for PAR requests. Specifically, the `fapi-1-baseline` profile contains `pkce-enforcer` executor, which makes sure
+that client use PKCE with secured S256 algorithm. This is not required for FAPI Advanced clients unless they use PAR requests.
+
+In case you want to use <<_backchannel_authentication_endpoint,CIBA>> in a FAPI compliant way, make sure that your clients use both `fapi-1-advanced` and `fapi-ciba` client profiles.
+There is a need to use `fapi-1-advanced` profile, or other client profile containing the requested executors, as `fapi-ciba` profile contains just CIBA-specific executors.
+When for enforcing the requirements of the FAPI CIBA specification, there is a need for more requirements, like enforcement of confidential clients or certificate-bound access tokens.
+
+==== Open Banking Brasil Financial-grade API Security Profile
+
+{project_name} is compliant with the https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-financial-api-1_ID1.html[Open Banking Brasil Financial-grade API Security Profile 1.0 Implementers Draft 1].
+This one is more strict in some requirements than the <<_fapi-support,FAPI 1 Advanced>> specification and hence it may be needed to configure link:{adminguide_link}#_client_policies[Client Policies]
+in the more strict way to enforce some of the requirements. Especially:
+
+* If your client does not use PAR, make sure that it uses encrypted OIDC request objects. This can be achieved by using client profile with `secure-request-object` executor configured with `Encryption Required` enabled.
+* Make sure that for JWS, client uses `PS256` algorithm. For JWE, client shall use `RSA-OAEP` with `A256GCM`. This may need to be set in all the link:{adminguide_link}#_oidc_clients[Client Settings] where these algorithms are applicable.
```

Comment:
Make it
"the client uses the `PS256` algorithm. For JWE, the client should use the `RSA-OAEP`"